esmvalcore/_data_finder.py

```diff
@@ -145,21 +145,17 @@ def _apply_caps(original, lower, upper):
 
 def _resolve_latestversion(dirname_template):
     """Resolve the 'latestversion' tag."""
-    if '{latestversion}' not in dirname_template:
-        return dirname_template
-
-    # Find latest version
-    part1, part2 = dirname_template.split('{latestversion}')
-    part2 = part2.lstrip(os.sep)
-    if os.path.exists(part1):
-        versions = os.listdir(part1)
-        versions.sort(reverse=True)
-        for version in ['latest'] + versions:
-            dirname = os.path.join(part1, version, part2)
-            if os.path.isdir(dirname):
-                return dirname
-
-    return dirname_template
+    versions = glob.glob(dirname_template)  # -> list
```

Comment:
this is globbing on eg `/badc/cmip5/data/cmip5/output1/` at the very beginning - the entire CMIP5 dir on BADC, talk that takes hours